### DISCUSSION TITLE
Call for backup! (GENERAL) [U]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#23] [Card Balance] [answered]
Kinsmen Unite! (IMPERIAL) [U]
Play on your combatant and pay 10 [DP] .The Warrior adds the base [F] , [S] , [A] and [V] of all other Imperial Warriors in his squad to its own base ratings. Its total [V] will not exceed 20.

Call For Backup! (GENERAL) [U]
Play on your defending combatant. For every 7 [DP] you spend, add the [F] and [S] of a different Warrior in your Squad to your combatant's.
:arrow_right: add also [A] and [V].

**Comment by TomekPL-1** [comment, marked as answer]
Call For Backup! (GENERAL) [U]
Play when your warrior is defending in combat. You can add NON UNIQUE warrior from your hand in to your SQUAD. You need to pay his V in DP , During this combat add his [F] and [S] to your combatant's. If your warrior has been wounded during this combat wound both warriors.

**Reply by 21-ric** [reply]
Woaw it could be a good card. I like this suggestion.

**Reply by 21-ric** [reply]
I mark this suggestion as answer. Let me know if you are not agree.
